src/current/v25.2/create-policy.md

```diff
@@ -53,20 +53,25 @@ The `USING` and `WITH CHECK` expressions can reference table columns and use ses
+| `SELECT`                             | ✓                                                   | —                                           | —                                                | —                                                 | —                                                     | —                                             |
+| `SELECT ... FOR UPDATE / FOR SHARE`  | ✓                                                   | —                                           | —                                                | ✓                                                 | —                                                     | —                                             |
+| `INSERT`                             | —                                                   | ✓                                           | ✓                                                | —                                                 | —                                                     | —                                             |
+| `INSERT ... RETURNING`               | —                                                   | ✓(a)                                        | ✓                                                | —                                                 | —                                                     | —                                             |
+| `UPDATE`                             | ✓                                                   | ✓(a)                                        | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `DELETE`                             | ✓                                                   | —                                           | —                                                | —                                                 | —                                                     | ✓                                             |
+| `INSERT ... ON CONFLICT DO UPDATE`   | ✓(b)                                                | ✓(b)                                        | —                                                | ✓                                                 | ✓                                                     | —                                             |
+| `UPSERT`                             | ✓(b)                                                | ✓(b)                                        | —                                                | ✓                                                 | ✓                                                     | —                                             |
+
+- ✓: Always applied when fetching an existing row for `UPDATE` or `DELETE` (CockroachDB uses the `USING` policy to filter rows before modifying or deleting).
```

Comment:
We use the ✓ in cells for SQL statements that aren't UPDATE or DELETE. Also, it's used in cells for the WITH CHECK expression. I wonder if this should just say something like 'Always applied', without going into detail about the SQL statements and expression type.